src/components/staking/PageValidators.vue

```diff
@@ -32,7 +32,7 @@
       >
         No results for these search terms
       </div>
-      <TmDataLoading v-if="$apollo.loading" />
+      <TmDataLoading v-if="$apollo.loading && !searchTerm" />
```

Comment:
Nice and simple!
I will accept the solution but want to share that there might be an issue in the future:
Imagine a slow connection, validators are still loading but the user is already searching. The loader would disappear. Therefor it would be better to actually set a `loaded=true` after the validators are loaded in the `result` hook. Check PageValidator